.github/workflows/cd.yml

```
@@ -0,0 +1,38 @@
+  push:
+    branches: [main, develop, cd-testing, cd-settings]
+  pull_request:
+    branches: [main, develop, cd-testing, cd-settings]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Debug - Show trigger info
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)